.circleci/config.yml

```diff
@@ -20,6 +23,8 @@ test: &test
     - run:
         name: Run Zunit
         command: |
+          export LC_ALL=C.UTF-8
+          export LANG=C.UTF-8
```

Comment:
why is this needed? Possibly place it as a comment as it does not seem obvious

Reply:
I don't know exactly why, but the zunit in circleci raised this error blow when it creates pipenv though it works well in the local.
See [here](https://click.palletsprojects.com/en/7.x/python3/)
```
  Creating test pipenv
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.4/dist-packages/pipenv/vendor/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/pipenv/vendor/click/core.py", line 676, in main
    _verify_python3_env()
  File "/usr/local/lib/python3.4/dist-packages/pipenv/vendor/click/_unicodefun.py", line 118, in _verify_python3_env
    'for mitigation steps.' + extra)
RuntimeError: Click will abort further execution because Python 3 was configured to use ASCII as encoding for the environment.  Consult http://click.pocoo.org/python3/for mitigation steps.

This system supports the C.UTF-8 locale which is recommended.
You might be able to resolve your issue by exporting the
following environment variables:

    export LC_ALL=C.UTF-8
    export LANG=C.UTF-8
```

Reply:
Seems like pipenv requires these variables to be set :) (It uses Click to parse the cli arguments).

Please put a comment like
```
#  Required by pipenv to work correctly
```

Reply:
Done.

Reply:
Looks good thanks